Framework/Kernel/src/PropertyManager.cpp

```diff
@@ -500,7 +501,12 @@ ::Json::Value PropertyManager::asJson(bool withDefaultValues) const {
   const size_t count = propertyCount();
   for (size_t i = 0; i < count; ++i) {
     Property *p = getPointerToPropertyOrdinal(static_cast<int>(i));
```

Comment:
(linting issue) Given that `getPointerToPropertyOrdinal` accepts an `int`, shouldn't the loop variable type be `int` and `count` be cast outside of the loop?